compiler/Dockerfile

```diff
@@ -23,4 +23,9 @@ RUN go mod download
 WORKDIR /go/src/github.com/vmware-tanzu/graph-framework-for-microservices/compiler
 RUN go mod download
 
+WORKDIR /go/src/github.com/vmware-tanzu/graph-framework-for-microservices/install-validator
```

Comment:
can we move this above existing `WORKDIR /go/src/github.com/vmware-tanzu/graph-framework-for-microservices/compiler
RUN go mod download`
and remove the next WORKDIR switch